docs/account_nonce.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Account nonce
+hide_title: false
+sidebar_position: 5
+description: Subspace account nonce
+keywords:
+    - account
+    - nonce
+last_update:
+  date: 03/07/2025
+  author: Ning Lin
+---
+The nonce is an integer value in the account data. The nonce is attached to the transaction to be signed and is increased by one for every transaction that is included on-chain. A signed transaction that contains a nonce that doesn't match the nonce in the account data will be rejected to include on-chain, this essentially prevents replaying of a previous on-chain signed transaction.
```

Comment:
Is the check "doesn't match the current nonce" or "must be greater than or equal to the current nonce"?

Reply:
"doesn't match the current nonce" is the check for a transaction to be included in the block and executed, by "match" I mean "equal to"

"must be greater than or equal to the current nonce" is the check for a transaction to be added to the tx pool.